custom/icds_reports/templates/icds_reports/icds_app/filters.html

```diff
@@ -1,3 +1,3 @@
 <location-filter class="col-md-4"  style="float: right !important;" ng-show="$ctrl.filters.indexOf('location_id') === -1" is-open-modal='$ctrl.isOpenModal' selected-location-id="$ctrl.data.location_id" selected-locations="$ctrl.selectedLocations"></location-filter>
-<additional-filter class="col-md-4"  style="float: right !important;" ng-show="$ctrl.filters.indexOf('gender') === -1 || $ctrl.filters.indexOf('age') === -1" filters="$ctrl.filters"></additional-filter>
+<additional-filter class="col-md-4"  style="float: right !important;" ng-show="$ctrl.filters.indexOf('gender') === -1 || $ctrl.filters.indexOf('age') === -1 || $ctrl.filters.indexOf('ageServiceDeliveryDashboard') === -1" filters="$ctrl.filters"></additional-filter>
```

Comment:
What does this bit do?

Reply:
This checks if the additional filter box should be displayed based on the values in `vm.filters`.

Reply:
Does that mean that anyone who writes any future report UI is going to have to know to add this (and maybe other filters) to that variable or it will show up?

Reply:
Well yes, and I think that it would be worth refactoring.